pkg/objstore/inmem.go

```diff
@@ -26,6 +26,14 @@ type InMemBucket struct {
 	attrs   map[string]ObjectAttributes
 }
 
+// ObjectWithAttrs is a struct defines object name, content and attributes.
+// Can be used together with NewInMemBucketWithObjectsAndAttrs for mocking.
+type ObjectWithAttrs struct {
```

Comment:
Do we need this? We could just add pass a map[string][]byte and build attributes automatically, no? (:

Reply:
Sorry, I don't understand the point. What do you mean by `build attributes automatically`? I need to mock the time, so passing attributes is necessary I think,

Reply:
Umm, if you were talking about the case that lastModified time is added in meta.json, then it is good to just pass `[]byte` here.